Mathlib/MeasureTheory/Function/LpSpace.lean

```diff
@@ -762,6 +762,24 @@ def indicatorConstLp (p : ℝ≥0∞) (hs : MeasurableSet s) (hμs : μ s ≠ 
   Memℒp.toLp (s.indicator fun _ => c) (memℒp_indicator_const p hs c (Or.inr hμs))
 #align measure_theory.indicator_const_Lp MeasureTheory.indicatorConstLp
 
+/-- A version of `Set.indicator_add` for `MeasureTheory.indicatorConstLp`.-/
+theorem indicatorConstLp_add (hμs : μ s ≠ ∞) (c' : E) :
```

Comment:
```suggestion
theorem indicatorConstLp_add {c' : E} :
```

`hμs` is already in the variables, and `c'` should be implicit since `c` is. Same below.